clone_twitter/user/serializers.py

```diff
@@ -166,7 +173,7 @@ class Meta:
         )
 
     def get_follows_me(self, follow):
-        me = self.context['me']
+        me = self.context['request'].user
```

Comment:
passed my user's pk in context but changed to pass request (considering tweet ... etc implementation style)